sources/packages/backend/apps/api/src/route-controllers/application-exception/application-exception.institutions.controller.ts

```diff
@@ -29,20 +29,22 @@ export class ApplicationExceptionInstitutionsController extends BaseController {
   /**
    * Get a student application exception of a student.
    * @param studentId student id.
+   * @param applicationId application id.
    * @param exceptionId exception to be retrieved.
    * @returns student application exception information.
    */
-  @Get("student/:studentId/exception/:exceptionId")
+  @Get("student/:studentId/application/:applicationId/exception/:exceptionId")
```

Comment:
Not a part of this PR but these parameters are too important for the authorization and I believe that they must be added as const like `APPLICATION_ID_PARAM` to be used all over the places.
```ts
@HasStudentDataAccess("studentId", APPLICATION_ID_PARAM)
```
```ts
@Get("student/:studentId/application/:`${APPLICATION_ID_PARAM}`/exception/:exceptionId")
```
Same for `studentId` and `locationId`.
Thoughts?

Reply:
Sounds good for me.

Reply:
as discussed, keeping as it is in this PR